concent_api/utils/helpers.py

```diff
@@ -154,6 +154,10 @@ def get_storage_file_path(subtask_id, task_id):
     return f'blender/result/{task_id}/{task_id}.{subtask_id}.zip'
 
 
+def get_source_package_path(subtask_id, task_id):
+    return f'blender/source/{task_id}/{task_id}.{subtask_id}.zip'
```

Comment:
This is not something that should be in `utils`. `utils` is for generic stuff not related specifically to Concent. This is part of the protocol or at least business logic. It has no place here. Please move it to one of the more specific apps.